src/test/java/org/folio/rest/support/matchers/PostgresErrorMessageMatchers.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.rest.support.matchers;
+
+import org.hamcrest.Matcher;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Is it fine to use this annotation? I think guys who uses another IDE may have some issues with this.

Reply:
That is a good question. I don't think it is IDE specific, I'm happy to remove it.